index.md

```diff
@@ -238,3 +238,67 @@ Our typical usages of AWS include:
 1. S3 for hosting uploaded files
 1. ElastiCache instances for hosting in-memory data stores
 1. OpsWorks for provisioning each if the size or complexity is large
+
+# Documentation
+
+Traditionally documentation has been a second-class citizen in most Rails project. This holds true just as much outside The Nerdery as internally. As innovators and leaders in our industry we should hold ourselves to a higher standard and therefore provide ample documentation on the products we produce. A great article on improving Rails based documentation published by Aaron Sumner can be found [here](http://everydayrails.com/2016/04/18/rails-documentation-practices.html) and many of the practices described below are pulled from his philosophy.
```

Comment:
In this case I've left it out because I've heard lots of mixed (mostly negatively) comments in regards to Swagger when maintained via Rails. I personally love Swagger, but my exposure to it has been centered around PHP work. When working with Symfony you are able to annotate docblocks and have the swagger docs automatically generated for you. It's extremely easy an intuitive. Based on my research and rumblings I don't think that's the case with Rails and therefore don't want to force us into that box.

So what's the alternative? Well, the beauty with YARD is we can define these tags as we get a feel for it to address what most of those Swagger things address. Obviously Swagger will still hit on things we can't with Swagger, but should do most of it:

    /**
     * Really bad arbitrary example just to give someone an idea
     *
     * @get
     * @route /heres/some/route
     * @returns {json}
     */ 

Explanation aside, this is what I gathered off of my research and talking with you all. Swagger is the de-factor standard of API documentation development and personally I would like to go that route, but I don't want to force everyone to use it if it's an absolute headache. If team has different feelings about this and feels we should agree on using Swagger, or some other API documentation tech. Let's discuss it.